cmd/build/v2/build_test.go

```diff
@@ -249,15 +260,6 @@ func TestValidateOptions(t *testing.T) {
 
 func TestOnlyInjectVolumeMountsInOkteto(t *testing.T) {
 	ctx := context.Background()
-	okteto.CurrentStore = &okteto.OktetoContextStore{
```

Comment:
Should we remove all occurences of `okteto.CurrentStore` in the `build` pkg?

Reply:
done 2ea04fc0bb6661fe3d92345f7270effde1bf0290  c5801b6e8a28749b6d09cba17210e2b0f2f962ba